docs/UserGuide.md

```diff
@@ -70,23 +71,22 @@ As long as you need to keep track of upcoming events or class, you are our audie
 or above installed in your Computer.
 1.  Download the latest Pac.jar from [here](https://github.com/AY1920S2-CS2113T-T12-4/tp/releases).
 1.  Copy the file to the folder you want to use as the home folder for this application.
-1.  Type `java -jar Pac.jar` to start the application.
-    <INSERT SCREENSHOT>
-1.  You should see this screen if everything is successful.
 
 ### 2.2 Startup using Command Line
 1.  Open your terminal.
 1.  Navigate to the home folder containing Pac.
 1.  cd followed by the file path into the terminal as shown below
     <INSERT IMAGE> 
 1.  Type `java -jar` followed by the name of the jar file and press **Enter**.
+1.  You should see this screen if everything is successful.
```

Comment:
Is there any picture added after this line?

Reply:
yes. I'll add it now